CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Ability to run a subflow as if it were an operation using the
   `dffml.dataflow.run` operation.
 - Support for operations without inputs.
+- Partial doctestable examples to `features.py`
+
```

Comment:
```suggestion
```

We want to maintain consistency with the rest of the sections by not adding a blank line here